test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -237,7 +237,7 @@ public class QuoteInput
             public float ID;
 
             [Column("1")]
-            public string Text;
+            public ReadOnlyMemory<char> Text;
```

Comment:
We still support regular `string` types, right?  I think that would be bad if we didn't.